src/blocks/custom.rs

```diff
@@ -97,6 +108,18 @@ impl Block for Custom {
 
         self.output.set_text(output);
 
+        if let Some(state_command) = self.state_command.clone() {
+            let state = Command::new(env::var("SHELL").unwrap_or("sh".to_owned()))
+                .args(&["-c", &state_command.clone()])
+                .output()
```

Comment:
What happens if the command fails?

Reply:
Described it in https://github.com/greshake/i3status-rust/pull/454#discussion_r336893795